front/src/modules/ui/table/components/EntityTableDragSelect.tsx

```diff
@@ -0,0 +1,65 @@
+import { RefObject } from 'react';
+import {
+  boxesIntersect,
+  useSelectionContainer,
+} from '@air/react-drag-to-select';
+import { useRecoilCallback, useRecoilValue } from 'recoil';
+
+import { isRowSelectedFamilyState } from '../states/isRowSelectedFamilyState';
+import { tableRowIdsState } from '../states/tableRowIdsState';
+
+type OwnProps = {
+  tableBodyRef: RefObject<HTMLDivElement>;
```

Comment:
can we abstract this a little bit more to uncouple from table:
- dragSelectable: ref
- onDragSelect: (id: string) => void

selectable would be used to:
- set eventsElement to be dragSelectable itself
- compute the list of selectable items: all selectable children that have a 'selectable-id' attribute

onDragSelect would be called with the selectable-id as attribute.

We can also find a more generic name for EntityTableDragSelect. Maybe just DragSelect and move this into @/ui/utils/drag-select

Then we can use it from EntityTable